data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,397 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto reading
+
+			label teeneep2
+			action
+				log `A Hai intelligence report explained her frequent visits to the Unfettered began in he early life due to her Unfettered grandfather. This led to her fencing skill, and victories in the Hai equivalent of the Olympics. Later, Sayari encouraged her to ride her fame into politics, forming a pro-rights and anti-corruption movement that angered Elders, often attacking them "by name."`
+				set "Hai Reveal [A11] Sound the Alarm: teeneep2 book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	Teeneep's rise to fame began, it seems, on Unfettered worlds, due to her superb fencing skills. Her grandfather is a well-known duelist and mid-ranked Unfettered officer, which ensured Teeneep a warm welcome among the Unfettered. She was known for rarely losing fights, even against opponents twice her weight or triple her age. This fame granted her entry into Hai athletics; they have an equivalent to the Olympics, which includes all Hai worlds, even the Unfettered. There, her fame rapidly grew to the point where "Teeneep" was a household name among all Hai. The report says she was still, "barely an adult," and that the "risks of giving social power to someone so young became obvious."`
+			`	She rode her fame into politics after befriending Sayari, who encouraged Teeneep to pursue her childhood dreams of helping the underprivileged. Teeneep led a movement that united thousands of local activist organizations into a massive political force. They pushed for fair treatment of humans in Hai space, safety of Hai in human space, increased intake of human refugees, investigations into disappearances, anti-corruption laws, and other measures, all directed at perceived unfairness in society. The report pointedly lists over five hundred instances where "Teeneep's movement publicly attacked the actions of Elders 'by name'."`
+			`	It warns the growing movement has become a threat to the stability of both Hai and human society.`
+				goto reading
+
+			label quarg
+			action
+				log `A Hai intelligence report reports a mutual fondness between Hai and Quarg for one anothers' cuisine. It also mentioned lingering frustration about the Quarg refusing to fight the Korath during their warlike spiral that led to near extinction. The Hai felt that Quarg involvement would have saved more Korath lives.`
+				set "Hai Reveal [A11] Sound the Alarm: quarg book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as, "subtle and mysterious."`
```

Comment:
```suggestion
			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as "subtle and mysterious."`
```